core/tests/upgrade-test/tests/upgrade.test.ts

```diff
@@ -280,9 +280,11 @@ describe('Upgrade test', function () {
         );
         executeOperation = chainUpgradeCalldata;
 
+        console.log('Sending scheduleTransparentOperation');
```

Comment:
Motivated by this test [sometimes failing](https://github.com/matter-labs/zksync-era/actions/runs/10715375714/job/29717077740?pr=2804) with "nonce has already been used" error on one of the operations below.